docs/architecture/blazor-for-web-forms-developers/architecture-comparison.md

```diff
@@ -12,7 +12,15 @@ While ASP.NET Web Forms and Blazor have many similar concepts, they also have di
 
 ## ASP.NET Web Forms
 
-TODO
+ASP.NET Web Forms are built with a page-centric architecture where each request for a location in the application is a separate page that ASP.NET replies with.  As pages are requested, the content of the browser are completely replaced with the results of the page requested.
+
+Pages are made up of HTML markup, C# or Visual Basic Code, a code-behind class that provides logic and event-handling capabilities, and controls.  Controls are reusable units of web user-interface that can be programmatically placed and interacted with on a page.  Pages are made up of files that end with .ASPX containing markup, controls, and some code.  The code-behind classes reside in files with the same base name and an .ASPX.CS or .ASPX.VB extension depending on the programming language used for the code-behind file.  Interestingly, the ASPX file contents are interpreted by the web server and re-compiled whenever they change, even while the web server is already running.
```

Comment:
I think we're using lowercase when refering to file extensions elsewhere. Let's be consistent. I believe the docs also typically put file names in *italics*, but @scottaddie should confirm.

Reply:
You're correct.